specs/sharding/beacon-chain.md

```diff
@@ -94,6 +99,7 @@ The following values are (non-configurable) constants used throughout the specif
 | `INITIAL_ACTIVE_SHARDS` | `uint64(2**6)` (= 64) | Initial shard count |
 | `GASPRICE_ADJUSTMENT_COEFFICIENT` | `uint64(2**3)` (= 8) | Gasprice may decrease/increase by at most exp(1 / this value) *per epoch* |
 | `MAX_SHARD_HEADERS_PER_SHARD` | `4` | |
+| `MAX_SHARD_PROPOSER_SLASHINGS` | `2**6` (= 64) | Maximum amount of shard proposer slashing operations per block |
```

Comment:
Interesting... to account for one slashing per shard per slot.

I think I'd argue that the complexity of verification and size of this is similiar to beacon block and so should maybe be 16 instead. Obviously you have issues if all shards are slashable all the time, but in expectation you will be able to catch up with 16 over time to some extent.

Numbers aren't amazing on the catchup in the extreme though

Reply:
Adjusted the config vars for now. We can revisit this later, there's something to say for more (more slashings are less of a DoS danger, only bad if there's also a lot of forking)